### ISSUE TITLE
[Accessibility] Carousel block - Dots navigation doesn't include `aria-current="true"` for current item/slide

### ISSUE THREAD

**Issue by andeng1106** [gambitph/Stackable#3250]
**Describe the bug**
- If current slide is present, the dot navigation for the Carousel block doesn't have an `aria-current="true"`
- See more info [here](https://ej2.syncfusion.com/react/documentation/carousel/accessibility#aria-attributes:~:text=all%20indicator%20elements.-,aria%2Dcurrent,-For%20the%20active)

**To Reproduce**
1. Add a new page
2. Add a Carousel block
3. Save page and view frontend
4. Inspect Carousel's dot navigation (current page)
5. See that there's no `aria-current="true"` present

**Expected behavior**
- `aria-current="true"` should be visible only for the current slider present
- Here's a [sample video](https://drive.google.com/file/d/1EVc5zp954orURtnraEFJtMvQ_l6sVbSe/view?usp=drive_link) on how it looks like when navigating.

**Screenshots**

https://github.com/user-attachments/assets/344fb969-7ba1-4db6-a6ad-b9d832f3a52b
